packages/bpk-component-breadcrumb/readme.md

```diff
@@ -0,0 +1,50 @@
+# bpk-component-breadcrumb
+
+> Backpack breadcrumb component.
+
+## Installation
+
+```sh
+npm install bpk-component-breadcrumb --save-dev
+```
+
+## Usage
+```js
+import React, { Component } from 'react';
+import BpkBreadcrumb, { BpkBreadcrumbItem } 'bpk-component-breadcrumb';
+
+
+export default class App extends Component {
+  render() {
+    return (
+      <BpkBreadcrumb label="Default breadcrumb">
+        <BpkBreadcrumbItem href="/">Home</BpkBreadcrumbItem>
+        <BpkBreadcrumbItem href="/page-1">Page 1</BpkBreadcrumbItem>
+        <BpkBreadcrumbItem active aria-current="page">
+          Page 2
+        </BpkBreadcrumbItem>
+      </BpkBreadcrumb>
+    );
+  }
+}
+
+```
+## Props
+
+### BpkBreadcrumb
+
+| Property           | PropType | Required | Default Value |
+| ------------------ | -------- | -------- | ------------- |
+| children           | node     | true     | -             |
+| label              | string   | true     | -             |
+
+
+### BpkBreadcrumbItem
+
+| Property           | PropType                   | Required | Default Value |
+| ------------------ | -------------------------- | -------- | ------------- |
+| children           | node                       | true     | -             |
+| href               | string                     | true     | null          |
+| aria-current       | string                     | true     | page          |
+| active             | bool                       | false    | false         |
+| linkProps          | object                     | false    | null          |
```

Comment:
In the code the default value for `linkProps` is `{}`, but here it's `null`. One of them should be changed so that they match.